### DISCUSSION TITLE
Install/run the Congo Doc site locally?

### DISCUSSION THREAD

**Discussion by harperjohn** [jpanther/congo#142] [Q&A]
Hi 

I am in an "air-gapped" development environment.

Can the doc website be run locally?

Thanks!!!

Looking forward to learning Hugo and using the Congo theme!!!

**Comment by jpanther** [comment]
Yes, it's possible to run the site locally. Just clone the repo, then install dependencies and run `npm run example`. By default, the site will be available at http://localhost:8008/congo/.

**Reply by harperjohn** [reply]
Thanks for getting back  to me!

I  cloned and did a "npm install"

and then the "npm run example"

congo > npm run example

> hugo-congo-theme@2.0.5 example
> hugo server --source exampleSite --themesDir ../.. --buildDrafts -b http://localhost/congo/ -p 8008

Start building sites …
hugo v0.88.1-5BC54738 windows/amd64 BuildDate=2021-09-04T09:39:19Z VendorInfo=gohugoio
ERROR 2022/03/05 19:36:04 Failed to get JSON resource "https://api.twitter.com/1/statuses/oembed.json?id=&omit_script=true": Failed to retrieve remote file: Not Found, body: "{\"errors\":[{\"message\":\"Sorry, that page does not exist\",\"code\":34}]}"
If you feel that this should not be logged as an ERROR, you can ignore it by adding this to your site config:
ignoreErrors = ["error-remote-getjson"]
Error: Error building site: logged 1 error(s)
Built in 4052 ms

**Reply by jpanther** [reply]
Hmm, that looks like a Twitter API error. I wonder if this is something that could be fixed by updating your Hugo version? I don't see this error but I'm using Hugo 0.92.2 in my testing.

**Reply by harperjohn** [reply]
That was it!!! 

I installed the latest release, (Hugo 0.93.2) and it seemed to work fine!!! 

Thanks again!!
